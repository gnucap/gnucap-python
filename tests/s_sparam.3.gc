attach public c_python.so

loadpy s_sparam.py


spice

.verilog

sparam_port #(.impedance(10)) p1(nin 0);
sparam_port #(.impedance(10)) p2(nout 0);

resistor #(.r(1)) rin(nin, nm)
resistor #(.r(1)) rgi(nm, 0)

resistor #(.r(1)) rout(nm, nout)
capacitor #(.c(0.1)) rout(nin, nout)

vcvs #(.gain(1)) e1(nc, 0, nin, 0)
resistor #(.r(1)) rc(nc, nout)

list

options numdgt 1

op
sp paramtype=s port p* 1 16 * 2
sp paramtype=y port p* 1 4 * 2
sp paramtype=z port p* 1 4 * 2


end
