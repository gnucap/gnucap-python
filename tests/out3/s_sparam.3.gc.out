Gnucap : The Gnu Circuit Analysis Package
Never trust any version less than 1.0
Copyright 1982-2013, Albert Davis
Gnucap comes with ABSOLUTELY NO WARRANTY
This is free software, and you are welcome
to redistribute it under the terms of 
the GNU General Public License, version 3 or later.
See the file "COPYING" for details.
main version
core-lib version
default plugins
generator: already installed, replacing
stashing as generator:0
transient: already installed, replacing
stashing as transient:0
transient: already installed, replacing
stashing as transient:0
PAC #(.impedance(10)) p1 (.p(nin),.n(0));
PAC #(.impedance(10)) p2 (.p(nout),.n(0));
resistor #(.r(1)) rin (.p(nin),.n(nm));
resistor #(.r(1)) rgi (.p(nm),.n(0));
resistor #(.r(1)) rout (.p(nm),.n(nout));
capacitor #(.c(0.1)) rout (.p(nin),.n(nout));
vcvs #(.gain(1)) e1 (.p(nc),.n(0),.ps(nin),.ns(0));
resistor #(.r(1)) rc (.p(nc),.n(nout));
#        
 27.     
#@freq   
. 1.
-0.731   - 0.0342  * i 0.639   - 0.117   * i
 0.213   - 0.0059* i-0.39- 0.078* i
. 2.
-0.757   - 0.0448  * i 0.554   - 0.111   * i
 0.208   - 0.0094* i-0.448   - 0.0823  * i
. 4.
-0.785   - 0.0363  * i 0.508   - 0.0662  * i
 0.202   - 0.00854 * i-0.488   - 0.0555  * i
. 8.
-0.798   - 0.0213  * i 0.495   - 0.0336  * i
 0.198   - 0.00521 * i-0.503   - 0.0301  * i
. 16.
-0.802   - 0.0111  * i 0.493   - 0.0168  * i
 0.197   - 0.00275 * i-0.507   - 0.0154  * i
#@freq   
. 1.
Y incomplete
. 2.
Y incomplete
. 4.
Y incomplete
#@freq   
. 1.
 2.22    - 0.45* i 1.86    - 0.225   * i
 5.49    - 1.55    * i 5.16    - 1.11    * i
. 2.
 1.89    - 0.487   * i 1.69    - 0.244   * i
 4.44    - 1.35    * i 4.38    - 1.04    * i
. 4.
 1.64    - 0.344   * i 1.57    - 0.172   * i
 3.92    - 0.781   * i 3.94    - 0.648   * i
